recipes/cppcheck/all/conandata.yml

```
@@ -1,11 +1,16 @@
 sources:
+  "2.8.0":
+    url: "https://github.com/danmar/cppcheck/archive/2.8.tar.gz"
+    sha256: "57298f3b805f0eb816a04115fbc70e701f75083cfb0305a44246e365cf27606a"
   "2.7.5":
     url: "https://github.com/danmar/cppcheck/archive/2.7.5.tar.gz"
     sha256: "6c7ac29e57fa8b3ac7be224510200e579d5a90217e2152591ef46ffc947d8f78"
   "2.7.4":
     url: "https://github.com/danmar/cppcheck/archive/2.7.4.tar.gz"
     sha256: "f0558c497b7807763325f3a821f1c72b743e5d888b037b8d32157dd07d6c26e1"
 patches:
+  "2.8.0":
+    - {}
```

```suggestion
  "2.8.0": []
```
I think it's what you are trying to do, but did you check that all the patches in 2.7.5 listed below have been integrated in cppcheck 2.8 ?

---

It is not the same, CMAKE_SOURCE_DIR points to [this file](https://github.com/conan-io/conan-center-index/blob/master/recipes/cppcheck/all/CMakeLists.txt) whereas PROJECT_SOURCE_DIR points to the CMakelists.txt inside cppcheck sources. this change must be done for cppcheck 2.8.0 too.